PopcornEditor/src/butter.js

```diff
@@ -922,67 +922,127 @@ window.Butter = {
           finishedCallback( project );
         }
 
+        function initialMediaSources( qs ){
```

Comment:
When originally creating the library, I just figured that any code would have to initialize the editor and then load in whatever video or project data it felt was applicable. As far as storing video information in the url, if we wanted multiple, they would definitely have to be shortened somehow (and we couldn't store project data). I don't really have any preferences as to how that should be implemented so far though.

Project data however is a fairly small blob of json which might be easier to deal with? I created a [utility function](https://github.com/mozilla/popcorn-editor/blob/master/PopcornEditor.js#L78) for creating a project json out of a video url